registry/storage/linkedblobstore.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/distribution/reference"
 	"github.com/google/uuid"
 	"github.com/opencontainers/go-digest"
+	godigest "github.com/opencontainers/go-digest"
```

Comment:
Why are you importing this package here when it's imported literally one line above?